GUIDELINES.adoc

```diff
@@ -33,7 +33,7 @@ Use the following template for glossary entries:
 * `\image:images/caution.png[with caution]`
 |`<term>`               |The term to be described.          |
 |`<class>`              |The part of speech of the term.|Possible values: noun, verb, adjective, adverb, pronoun, conjunction, preposition. Use noun for acronyms or abbreviations. Leave the field empty for symbols.
-|`<description>`        |The description of the term.|Use full sentences with periods. If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. Add information about why we should use the term in this form. If we should not use the term, explain why. Alternatively, explain why the terms listed in the *Incorrect forms* field are incorrect.
+|`<description>`        |The description of the term.|Use full sentences with periods. If a term is specific to a particular product, start the description with "In <product_name>,". If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. Add information about why we should use the term in this form. If we should not use the term, explain why. Alternatively, explain why the terms listed in the *Incorrect forms* field are incorrect.
```

Comment:
```suggestion
|`<description>`        |The description of the term.|Use full sentences with periods. If a term is specific to a particular product, start the description with "In <product_name>,". If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. Add information about why we should use the term in this form. Alternatively, explain why the terms listed in the *Incorrect forms* field are incorrect.
```

Reply:
@porueesq As far as I can tell this isn't a duplicate sentence. Can you explain which other sentence you think it duplicates?

The one I think you might be looking at is the last one about the Incorrect forms. If so, that one is talking about adding reasons why the values in the "Incorrect forms" field are incorrect. But this one is talking about if the "use it" field is set to no, explain why the term shouldn't be used. For example: https://redhat-documentation.github.io/supplementary-style-guide/#puppetize

Reply:
@bergerhoffer Thanks for the explanation. Then, this should be clarified in the text as well to clearly distinguish these cases: 
* why a term is used
* why the same term isn't used
* why incorrect forms of the original term are considered as such



```suggestion
|`<description>`  |The description of the term. |Use full sentences with periods. Add information about why we should use the term in this form. If a term is specific to a particular product, start the description with "In <product_name>,". If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. When the term in this form is not appropriate for usage, explain the reasons. Alternatively, explain why we consider the terms listed in the *Incorrect forms* field as erroneous.
```

Reply:
@porueesq I had updated this section per miriam's suggestion to be simpler. Can you let me know if that update was sufficient, or if you think we need to make these updates still, thanks!

Reply:
Sounds good. I'll approve the MR.